### DISCUSSION TITLE
Synch Social Network carddav data from Nextcloud to Android phones

### DISCUSSION THREAD

**Discussion by Isac99** [bitfireAT/vcard4android#39] [Ideas]
HI,
Is it possible to synch the Social Network carddav data from Nextcloud (28.0.5) Contacts App (5.5.3) to Android phones? Currently only websites synch.
Is this a limitation of Android?
Thanks in advance

**Comment by devvv4ever** [comment]
You can use our app DAVx5 to sync contacts between a CardDAV/CalDAV server like Nextcloud with your Android devices.

**Reply by Isac99** [reply]
Hi I know that, but the social network fields are not syncing.

**Reply by ArnyminerZ** [reply]
Hi, right now Nextcloud is passing the social profiles as `X-SOCIALPROFILE`, which is not an standard CardDAV property. There's a proposal for RFC-9554 which adds the [`X-SOCIALPROFILE`](https://www.ietf.org/rfc/rfc9554.html#name-socialprofile) option, but it's still a proposal, and not even matches the format used by Nextcloud.

Nextcloud uses:
```vcard
BEGIN:VCARD
VERSION:3.0
PRODID:-//Sabre//Sabre VObject 4.5.4//EN
UID:afbf5f5d-b30c-4f8c-b554-9018f508e7ac
FN:Contact with Social Networks
ADR;TYPE=HOME:;;;;;;
EMAIL;TYPE=HOME:
TEL;TYPE=HOME,VOICE:
X-SOCIALPROFILE;TYPE=NEXTCLOUD:https://example.com
REV;VALUE=DATE-AND-OR-TIME:20240515T143810Z
END:VCARD
```
See that it uses `TYPE`, but the proposed standard uses `SERVICE-TYPE`, so even if it gets approved at some time, Nextcloud would still have to update its source to be compatible.

In any case, I don't think it's even supported by Android. _Maybe_ it might work with [Social Stream text](https://developer.android.com/guide/topics/providers/contacts-provider#SocialStream), however I don't think it's the same.

Then, I'd say it's currently not supported by Android, then also probably not by any Contacts app. If you have any extra information don't doubt on contacting again and we will see if it can be made compatible.

Sorry for the inconvenience.

**Comment by Isac99** [comment]
Thank you for the information. Hopefully this can help others with the same question.
